sentry/src/main/java/io/sentry/util/Pair.java

```diff
@@ -0,0 +1,39 @@
+package io.sentry.util;
+
+import org.jetbrains.annotations.Nullable;
+
+public final class Pair<A, B> {
```

Comment:
can we make it package-private or mark it as internal

Reply:
Package private only if we move it out to root package, so i guess making internal is the way to go.